fuel-crypto/Cargo.toml

```diff
@@ -15,7 +15,7 @@ borrown = "0.1"
 coins-bip32 = { version = "0.8", default-features = false, optional = true }
 coins-bip39 = { version = "0.8", default-features = false, features = ["english"], optional = true }
 ecdsa = { version = "0.16", default-features = false }
-ed25519-dalek = { version = "1.0", default-features = false, features = ["u64_backend"] }
+ed25519-dalek = { version = "2.0.0", default-features = false }
```

Comment:
Is `u64_backend` a default behavior right now?

Reply:
I couldn't see what happened to it, so I figured I'd rely on the the compiler to tell me.